vllm/multimodal/inputs.py

```diff
@@ -158,10 +162,11 @@ def __eq__(self, other: object) -> bool:
         return nested_tensors_equal(self.is_embed, other.is_embed)
 
 
-NestedTensors = Union[list["NestedTensors"], list[torch.Tensor], torch.Tensor,
-                      tuple[torch.Tensor, ...]]
+NestedTensors: TypeAlias = Union[list["NestedTensors"], list[torch.Tensor],
+                                 torch.Tensor, tuple[torch.Tensor, ...]]
```

Comment:
```suggestion
NestedTensors: TypeAlias = Union[list["NestedTensors"], list["torch.Tensor"],
                                 "torch.Tensor", tuple["torch.Tensor", ...]]
```

Reply:
Is ruff supposed to catch these?

Reply:
updated

Reply:
No, because you lazy load torch here. Can you check if we use torch during runtime in this file anywhere?